parser/fuzz/fuzz_targets/structured.rs

```diff
@@ -0,0 +1,47 @@
+#![no_main]
+
+mod common {
+	include!(concat!(env!("OUT_DIR"), "/common.rs")); // from build.rs
+}
+
+use ezno_parser::{ASTNode, Module, ParseOutput, SourceId, ToStringSettingsAndData};
+use libfuzzer_sys::{fuzz_target, Corpus};
+use pretty_assertions::assert_eq;
+
+fn do_fuzz(data: common::FuzzSource) -> Corpus {
+	let input = data.source;
+
+	let Ok(ParseOutput(module, state)) = Module::from_string(
+		input.to_owned(),
+		Default::default(),
+		SourceId::NULL,
+		None,
+		Vec::new(),
+	) else {
+		return Corpus::Reject
+	};
+
+	let output1 =
+		module.to_string(&ToStringSettingsAndData(Default::default(), state.function_extractor));
+
+	let Ok(ParseOutput(module, state)) = Module::from_string(
+		output1.to_owned(),
+		Default::default(),
+		SourceId::NULL,
+		None,
+		Vec::new(),
+	) else {
+		panic!("input: `{input}`\noutput1: `{output1}`\n\nThis parse should not error because it was just parsed above");
+	};
```

Comment:
why was this altered? is expect not simpler?

Reply:
panic gets rid of the format. I assume this is style preference :slightly_smiling_face:

Reply:
Not just a style preference. Just a minor thing but (assuming it is not optimised out) the `format!` will eagerly allocate and interpolate a string even if the result is `Ok`. The latter only happens for `Err` so should be a bit faster 😉